deployments/helm/gcp-service-broker/templates/NOTES.txt

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.svccat.register }}
+
+The broker has been registered with the Kubernetes Service Catalog. The catalog
+will update its service and plan listings once the broker becomes healthy.
+
+You can check on its status by running:
+
+  kubectl describe clusterservicebroker {{.Values.svccat.name}}
+
+{{- else }}
```

Comment:
Is it possible for a service broker to be shared? Maybe that's a use case we won't support?